devdocs/process.md

```diff
@@ -118,8 +122,8 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
    * Push your branch to the committer repo (push to the fork if you do not 
      have push permission to the committer repo), if you haven't done that already.
    
-   * Create a pull request (PR). For the pull request name, copy paste the relevant
-     issue name.<br>
+   * <a name="creating-a-pull-request"></a>Create a pull request (PR). For the pull request name, copy paste
+     the relevant issue name and include the issue number as well.<br>
      e.g. ` Incorrect error message when adding an existing instructor #1760`<br>
      In the PR description, mention the issue number in this format: `Fixes #1760`. 
      Doing so will create an automatic reference from the issue to the pull request.<br>
```

Comment:
This is an inaccurate description, but not related to the PR. I'll open an issue for this.